internal/runners/manifest/version.go

```diff
@@ -0,0 +1,55 @@
+package manifest
+
+import (
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/pkg/platform/api/buildplanner/model"
+	platformModel "github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/ActiveState/cli/pkg/platform/runtime/artifact"
+)
+
+type resolvedVersion struct {
+	Requested string `json:"requested,omitempty"`
```

Comment:
```suggestion
	Requested string `json:"requested"`
```

I think it's friendlier to explicitly show an empty value.